tst/unit/test_output_utils.cpp

```diff
@@ -0,0 +1,281 @@
+//========================================================================================
+// Athena++ astrophysical MHD code
+// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+// Licensed under the 3-clause BSD License, see LICENSE file for details
```

Comment:
Is this still appropriate for the new file or should this be PArthenon?

Reply:
Agree. Fixed.